src/components/track-changes/suggestion-list/Suggestion.tsx

```diff
@@ -175,22 +176,40 @@ const Wrapper = styled.li<{
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  padding: ${(props) => props.theme.grid.unit * 3}px
+  gap: ${(props) => props.theme.grid.unit * 4}px;
+  padding: ${(props) => props.theme.grid.unit * 1.5}px
     ${(props) => props.theme.grid.unit * 2}px !important;
-  border-top: 1px solid ${(props) => props.theme.colors.background.secondary};
+  margin-bottom: 6px;
+  border: ${(props) =>
+    props.isFocused
+      ? `1px solid ${props.theme.colors.border.tracked?.active}`
+      : `1px solid ${props.theme.colors.border.tracked?.default}`};
+  box-shadow: ${(props) =>
+    props.isFocused
+      ? `-4px 0 0 0  ${props.theme.colors.border.tracked?.active}`
+      : `none`};
   list-style-type: none;
+  font-size: ${(props) => props.theme.font.size.small};
 
   /* FocusHandle should cover entire card: */
   position: relative;
+  color: ${(props) => props.theme.colors.text.primary};
   background: ${(props) =>
     props.isFocused
-      ? props.theme.colors.background.fifth + ' !important'
-      : 'transparent'};
+      ? props.theme.colors.background.tracked?.active + ' !important'
+      : props.theme.colors.background.tracked?.default};
+
+  ${(props) =>
+    props.isFocused
+      ? `${Actions} {
+        visibility: visible;
+      }`
+      : ''}
 
   &:hover {
-    background: ${(props) => props.theme.colors.background.fifth} !important;
+    background: ${(props) => props.theme.colors.background.tracked?.hover};
```

Comment:
Why is "tracked" optional? What are the circumstances under which it wouldn't exist? And if so wouldn't it require some handling because it would return 'undefined' which is not a valid css? Will styled-components take care of that?

Reply:
updated as not optional...